test/integration/targets/systemd/tasks/test_systemd_version.yml

```diff
@@ -8,4 +8,6 @@
     that:
       - ansible_systemd.version | int > 0
       - ansible_systemd.version is match('^[1-9][0-9][0-9]$')
-      - ansible_systemd.features | regex_search('(\\+|-)(PAM|AUDIT)') is not none
+      - ansible_systemd.full_version | string
```

Comment:
Why are you converting a string into a string? Also, post-datatagging this fails because we expect the entire expression to be a boolean and it's a string.

https://dev.azure.com/ansible/ansible/_build/results?buildId=146625&view=logs&j=264c847f-169f-56e8-5705-7131dc687c94&t=35553c59-85fa-5d19-e359-ff5d1c656cca&l=13639